tool/tctl/common/auth_command.go

```diff
@@ -656,6 +656,9 @@ Orapki binary was not found. Please create oracle wallet file manually by runnin
 orapki wallet create -wallet {{.walletDir}} -auto_login_only
 orapki wallet import_pkcs12 -wallet {{.walletDir}} -auto_login_only -pkcs12file {{.output}}.p12 -pkcs12pwd {{.password}}
 orapki wallet add -wallet {{.walletDir}} -trusted_cert -auto_login_only -cert {{.output}}.crt
+{{ else }}
```

Comment:
nit: no spaces used elsewhere:

```suggestion
{{else}}
```